### ISSUE TITLE
Fix broken localization test

### ISSUE THREAD

**Issue by taoeffect** [okTurtles/group-income#278]
### Problem

After updating dependencies the localization test in the frontend fails with:

```
  1) Frontend Test Localization Gathering Function Verify output of transform functions:
     AssertionError: expected Object {} to have property 'A test of sorts'
      at Assertion.fail (node_modules/should/cjs/should.js:258:17)
      at Assertion.value [as property] (node_modules/should/cjs/should.js:335:19)
      at Context.<anonymous> (test/frontend.js:318:33)
```

### Solution

- It's probably better to not use the hackish directory traversal stuff in `scripts/i18n.js`. Maybe investigate appropriating / modifying one of the plugins mentioned in #164.
- Fix the test and re-enable it (remove the `.skip`)

/cc @vijayee

**Comment by sandrina-p** [comment]
Closing this because this code doesn't exist anymore. Now we use Cypress and every assert related with text using translations has been working correctly.
